### ISSUE TITLE
[Feature Request] Not interactable progress dialog

### ISSUE THREAD

**Issue by Hoto-Cocoa** [ncruces/zenity#62]
I'm creating a windows application that act as like Kiosk. We want show progress dialog to user to show payment processing progress, However It should not be able close by user interaction.

However, Zenity's Progress dialog is doesn't have options to do that.

Adding option or type to do this requirement would be nice.

Thanks.

**Comment by ncruces** [comment]
Did you already try [`NoCancel`](https://pkg.go.dev/github.com/ncruces/zenity#NoCancel)?

**Comment by Hoto-Cocoa** [comment]
Yes, I tried. But It shows Close button on menu bar and shows disabled OK button.

**Comment by Hoto-Cocoa** [comment]
```go
zenity.Progress(zenity.Title("Payment"), zenity.NoCancel())
```

This code displays below dialog, And the close button works. I want remove close button and OK button.

![image](https://github.com/ncruces/zenity/assets/21969938/8e7ae293-ad37-4401-a03d-944cb5090731)

**Comment by ncruces** [comment]
I don't think that's supported by [`zenity`](https://help.gnome.org/users/zenity/stable/) which this tries to emulate.

But let me think if the extension makes sense. For starters, `AutoClose` could make the disabled OK button go away. I need to think through the close button (compare with other platforms).
